process/sig-lifecycle.md

```diff
@@ -1,13 +1,30 @@
 # Special Interest Group (SIG) Life Cycle
 
-Special Interest Groups are under the direct governance of a Working Group (WG) which can create and close them as necessary and appropriate. SIGs are expected to have a very specific goals and objectives at the time of their creation which may or may not define if they are terminable and is entirely dependent on the nature of the SIG.
+Special Interest Groups are under the direct governance of their reporting Working Group (WG) and are bound to achieving a very specific goal. These groups may be terminated upon completion of their designating tasking, continued for larger and ongoing efforts, or otherwise subject to the governance, structure, and termination policies of the WG they are under. The creation of a SIG must dictate the focus, intent, goals, and deliverables as appropriate.
 
-The lifecycle of SIGs is therefore minimal and as follows.
+It is expected that the primary output of a SIG is not software. If the primary output is software, the work should be organized as a [project](./project-lifecycle.md).
 
-## Active
+SIG process should be minimal with as much as possible delegated to the WG. However we do need minimal process to at least ensure we have an accurate list of all the active SIGs in the OpenSSF. This document uses "must" to describe what items are required and "should" to suggest items that should be strongly considered (but not required).
 
-* The SIG is actively pursuing its goal.
+## To become `Tentative`:
 
-## Inactive
+* This is the default state of a new SIG that is not yet active
 
-* The SIG is no longer active, either it has completed its mission or the mission was abandoned. At which state, the WG the SIG is under will determine a termination status and record it within the corresponding repository.
+## To become `Active`:
+
+* A SIG must have a repository (prefixed with `sig-`) that includes the current state of the SIG, lists the chairs of the SIG, and a statement of the focus, intent, goals, and/or deliverables of the group
```

Comment:
SLSA Positioning, Specification, and Tooling, as well as Vulnerability Autofix SIG do not have repositories today. If we like this item, we can go back and work with those SIGs to set that up.

Reply:
I don't think we should make it a requirement to have a repo for every SIG. I think it is perfectly fine for a SIG to simply have a page as part of their parent WG's repo.

Reply:
> Each SIG must have a documented landing page, where one can learn more about the SIG, how to get involved, etc. but that should only be a dedicated repo if the repo will be otherwise active beyond providing that landing page.

I am definitely open to this direction, but help me understand - what would that landing page be? Would it be part of the WG's repository's README? Or a separate page in the WG's repository? Or something else?

Reply:
Do note from https://github.com/ossf/tac/issues/162 we have a tentative list (@hythloda has [explicitly asked for feedback](https://github.com/ossf/tac/issues/162#issuecomment-1572589533)) on what we expect of Projects / SIGs today: https://docs.google.com/spreadsheets/d/140czMjqhe9YxLBTx4rWkjHxWEWqD_tVQ1s8l9Q2PE2g/edit#gid=821838029

Reply:
> Would it be part of the WG's repository's README? Or a separate page in the WG's repository?

Either of these make sense to me. I think the important thing is that since a SIG must have a 1:1 relationship with a WG (or the TAC), the repo for that WG can list SIGs it is responsible for, with this information.

Reply:
I have updated the text so SIG repositories are optional, and the information is listed on the governing body's (WG or TAC) README.